### DISCUSSION TITLE
How to get started using this repo

### DISCUSSION THREAD

**Discussion by patrickshas** [UniversalViewer/universalviewer#933] [General]
Hello,
Thank you for all your work on this amazing repo.
I am probably not looking at the right place:
Is there any instructions re getting started with the repo other than the `npm install universalviewer --save` command?
I am trying to use this repo in a localhost setting to get familiar with it's workings.
Also, are there any instructions relating to setting up a database to use with this repo?
Thanks in advance,
Best

**Comment by demiankatz** [comment]
@patrickshas, have you seen the [examples page in the wiki](https://github.com/UniversalViewer/universalviewer/wiki/UV-Examples)? This includes lots of code sandbox links showing different ways of hooking up and configuring UV, and it might be helpful for you.

When you ask about setting up a database, what do you mean? The UV is designed primarily (though not exclusively) to consume content in the IIIF manifest format. The best way to create/provide those manifests will depend a lot on your content and use case. If you can share more about what you're trying to do, we might be able to provide some more specific advice.

**Reply by patrickshas** [reply]
@demiankatz Thank you. Much appreciated.

**Comment by edsilv** [comment]
Please see https://github.com/UniversalViewer/universalviewer/blob/main/CONTRIBUTING.md

**Reply by patrickshas** [reply]
@edsilv Thank you. Much appreciated.
